### DISCUSSION TITLE
The Unicode and Character name no longer are displayed in the grid. What happened?

### DISCUSSION THREAD

**Discussion by htosports** [character-map-uwp/Character-Map-UWP#327] [Q&A] [answered]
I love this APP. However, just as I am recommending, I notice it no longer shows the name and unicode in the grid. I use this all the time. It seems maybe there was an updated version, and the options to display that no longer work - it seems to be in a mouseover tip instead now. 
This seems like a bug.

**Comment by JohnnyWestlake** [comment]
I broke it almost a month ago :') https://github.com/character-map-uwp/Character-Map-UWP/issues/326

Would have been fixed earlier but I've not had a lot of time to work on this and I was in the middle of adding a new feature.  A fix will hopefully be rolled out in a few days.

**Comment by JohnnyWestlake** [comment, marked as answer]
This has been fixed at the start of June
